src/pages/Checkout.vue

```diff
@@ -0,0 +1,406 @@
+<template>
+  <div class="container components-page" :class="{ 'mobile-view': mobileView }">
+    <div class="row">
+      <div class="col-12">
+        <div>
+          <router-link to="/">Components</router-link> &middot;
+          <router-link to="/checkout">Checkout</router-link>
+        </div>
+        <h1>Checkout</h1>
+      </div>
+    </div>
+
+    <div class="row overview-screen-row" :class="{ 'allow-wrap': mobileView }" v-if="!paymentScreen">
```

Comment:
In the future, what do you think about splitting larger pages into more components so that our code is more readable? Or would that not be very easy with Vue?